templates/unlink_clone.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<inkscape-extension translationdomain="inkstitch" xmlns="http://www.inkscape.org/namespace/inkscape/extension">
+    <name>Unlink Clone</name>
+    <id>org.{{ id_inkstitch }}.unlink_clone</id>
+    <param name="extension" type="string" gui-hidden="true">unlink_clone</param>
+    <effect>
+        <object-type>all</object-type>
+        <effects-menu>
+            <submenu name="{{ menu_inkstitch }}" translatable="no">
+                <submenu name="Tools: Clone" />
```

Comment:
I'm not sure about adding a new menu category with only one tool in it, but it probably doesn't belong at the top level or any of the other tool groups.

Reply:
Maybe it's a fill tool feature as it is all about applying the fill angle when you apply the clones?
Not sure as I didn't look through the code yet :D
Or it could be in `Edit` as well?

Reply:
`Edit` seems like a reasonable place to put it to me.

Reply:
That's where I put it, anyway. I suppose technically we might want it to have perfect parity with Inkscape's normal `unlink` and only resolve one level of clone instead of recursively unlinking all clones? Hmph...

Reply:
Maybe just as an option? I'd like if it was able to do them all at once and do that as the default behavior.

Reply:
It's now an option.